client/src/components/Tickets/TicketCreateForm.jsx

```diff
@@ -156,8 +162,8 @@ function TicketCreateForm({
                   id="attachments"
                   isSubmitted={isSubmitting}
                   description="Error reports and screenshots can be helpful for diagnostics"
-                  maxSizeMessage="Max File Size: 3MB"
-                  maxSize={3145728}
+                  maxSizeMessage={maxSizeMessage}
+                  maxSize={maxSize}
```

Comment:
We can include the defaults in the code as well, that way portals don't necessarily need to set `maxSizeMessage` or `maxSize` values for this to work.
```suggestion
                  maxSizeMessage={maxSizeMessage || "Max File Size: 3MB"}
                  maxSize={maxSize || 3145728}
```